src/Python/wcon/wcon_schema.json

```diff
@@ -41,23 +41,13 @@
 			"description": "Information about experimental conditions and software versions; critical for reproducibility across labs.",
 			"properties": {
 				"lab": {"type": "object"},
-				"who": {"type": "string"},
+				"who": {"oneOf": [
```

Comment:
Very good question.  The reason is that the PyPI packager wraps up everything in the `src/Python/wcon/` folder and zips it up and that becomes what's distributed when you install wcon via `pip install wcon`.  Therefore since the schema must be included in the package, it must be in the `src/Python/wcon/` folder.

Of course, the schema is not Python-specific and so I wanted it to be in the root directory, as it is.  But that leaves us in the somewhat awkward position of having two identical `wcon_schema.json` files.

I can tell you that at least in the Travis-CI testing I validate that both files are identical, so we'll never have a difference.